src/base/misc/fatfs_boot.S

```diff
@@ -87,9 +87,27 @@ write_msg_20:
 	int	$0x10
 	jmp	write_msg
 
-/*# 5 magic bytes (for IO.SYS compatibility) :-))*/
-	.byte	4, 3, 2, 1, 13
-text:
+	.align 2
+	.globl	boot_txfr
+boot_txfr:
+	.word	0x9090 /* start ofs */
+	.word	0x9090 /* start seg */
+	.word	0x9090 /* ax */
+	.word	0x9090 /* bx */
+	.word	0x9090 /* cx */
+	.word	0x9090 /* dx */
+	.word	0x9090 /* si */
+	.word	0x9090 /* di */
+	.word	0x9090 /* bp */
+	.byte	0x90   /* number of list entries */
+	.byte	0x90   /* drive */
+	.fill	0x10*4 /* list of i1342 blocks */
```

Comment:
Yep it should be a constant I think. i.e sizeof(i1342blk) * 4 blocks. Is there any way to reuse that info from the disks.h header?